crates/sui-framework/packages/move-stdlib/sources/vector.move

```diff
@@ -263,6 +263,14 @@ module std::vector {
         acc
     }
 
+    /// Concatenate the vectors of `v` into a single vector, keeping the order of the elements.
+    public macro fun concat<$T>($v: vector<vector<$T>>): vector<$T> {
```

Comment:
@tnowacki , please take a look

Reply:
@damirka @tzakian did we have this on the list? 

The idea looks fine to me, just wondering about the naming. 
Maybe `flatten`?

Reply:
`flatten` feels like a better/more idiomatic name to name. Usually when I think of `concat` I think of concatenating two vectors/collections together.

Reply:
I'm fine with either, so let's go with `flatten`.